_posts/2024-06-03-youll-regret-using-natural-keys.html

```diff
@@ -74,7 +74,7 @@ <h3 id="778bd217e62d4ffca25f487a39d34c1a">
         Yes, the number may uniquely identify a person, but the converse may not be true. A person may have more than one identification number. At least when time is a factor.
     </p>
     <p>
-        As an example, for technical-historical reasons, the Danish CPR number carries information (which keys shouldn't do), such as a person's date of birth and sex. Since 2014 a new law enables transsexual citizens to get a new CPR number that reflects their perceived gender. The consequence is that the same person may have more than one CPR number. Perhaps not more than one at the same time, but definitely two during a lifetime.
+        As an example, for technical-historical reasons, the Danish CPR number carries information (which keys shouldn't do), such as a person's date of birth and sex. Since 2014 a new law enables transgender citizens to get a new CPR number that reflects their perceived gender. The consequence is that the same person may have more than one CPR number. Perhaps not more than one at the same time, but definitely two during a lifetime.
```

Comment:
Please don't change my words, however misguided they may be.

I'm happy to take pull requests that fix obvious typos. This one, I realize, is different. It's not a typo, but rather a word that some people find sensitive.

Before we proceed, please be aware that I'm not a native English speaker, and that Danish (my native language) doesn't have two words for what may or may not be the same concept. Both _gender_ and _sex_ translates to _køn_. So I'm having a hard time telling the English terms apart.

I treat the blog as a long-lasting source of information, and in good academic transition, I find _transparency_ important. In this context, it means that it's important to me that readers can trust that the blog is a _publication_; that is, that once published, text doesn't just change without notification. (Like every other good publisher, I do consider corrections of obvious typos as a fair exception to that rule.)

When I _do_ need to change something (which happens) I explicitly use the `del` and `ins` HTML tags in order to indicate ex-post edits. Here's an example: [Composition Root](https://blog.ploeh.dk/2011/07/28/CompositionRoot/) (towards the end). I also have an open GitHub issue to make this clearer: #538.

Reply:
Removed this change as requested. Moved my comment below the one that arrived in the interval to maintain chain.